### PR TITLE
Task 955 Rewrite simple English wikipedia sentences in more sophisticated English

### PR DESCRIPTION
**Pull request by garyhlai**
Sorry about the separate PR -- I realize that a reverse style transfer would also be helpful for this project. I named it task 934 so that it's close to 933 as in #348 (let me know if I should change it to the latest number)

- Instructions for making simple English wikipedia sentences more sophisticated. Input is simple English wikipedia sentence, output is the aligned wikipedia sentence in normal English.

- Created based on the HuggingFace dataset "wiki_auto"

- Citation:
@inproceedings{acl/JiangMLZX20,
author = {Chao Jiang and
Mounica Maddela and
Wuwei Lan and
Yang Zhong and
Wei Xu},
editor = {Dan Jurafsky and
Joyce Chai and
Natalie Schluter and
Joel R. Tetreault},
title = {Neural {CRF} Model for Sentence Alignment in Text Simplification},
booktitle = {Proceedings of the 58th Annual Meeting of the Association for Computational
Linguistics, {ACL} 2020, Online, July 5-10, 2020},
pages = {7943--7960},
publisher = {Association for Computational Linguistics},
year = {2020},
url = {https://www.aclweb.org/anthology/2020.acl-main.709/}
}

### CODE REVIEW

**Comment by swarooprm** [comment]
@ghlai9665, please change to the latest task number (later on we will ensure to keep similar tasks together, but now for continuity lets use the latest task number)

**Comment by danyaljj** [comment]
@Sujan242 wondering if you can review this PR?

**Comment by Sujan242** [comment]
@danyaljj Looks good to me.
It is similar to #348 but the input and output have been swapped over all the instances.

**Comment by danyaljj** [comment]
@ghlai9665 the reverse direction of this task (#348l; a compression task) was relatively well-defined. 
However, I am a little hesitant about this task. It is not quite clear what it means to make a given text more "sophisticated". We need to be able to what the task is about in a language that is understandable to a lay person. 
Unless we can provide a better language definition of this task, I am not sure if we can include this task.

**Comment by garyhlai** [comment]
> @ghlai9665 the reverse direction of this task (#348l; a compression task) was relatively well-defined. However, I am a little hesitant about this task. It is not quite clear what it means to make a given text more "sophisticated". We need to be able to what the task is about in a language that is understandable to a lay person. Unless we can provide a better language definition of this task, I am not sure if we can include this task.

Good point @danyaljj . I agree "sophisticated" is not well-defined enough. I have changed the task definition -- it looks much better to me now than before.

**Comment by danyaljj** [comment]
Please make sure to set the `"Categories": [""]` and `"Domains": [""]` fields according to [the taxonomy presented here](https://github.com/allenai/natural-instructions-expansion/blob/master/doc/task-hierarchy.md).

**Comment by swarooprm** [comment]
This task looks good to me. 
We just need to keep it in the training spilt while designing experiments.

**Comment by swarooprm** [comment]
> This task looks good to me. We just need to keep it in the training spilt while designing experiments.

@danyaljj, can we proceed with this PR? I know there is no 1-1 mapping between input and output, but this task can increase diversity and we can keep it in the training-split. In fact, most open-ended tasks such as story generation fall under this category.

File: tasks/task955_wiki_auto_style_transfer.json [hunk 1]
```diff
@@ -0,0 +1,10604 @@
...
+    "Contributors": [
+        "Gary Haizhi Lai"
+    ],
+    "Source": [
+        "wiki_auto"
+    ],
+    "Categories": [
+        "Style Transfer"
+    ],
+    "Definition": "In this task, we ask you to elaborate the sentence without changing its general meaning. You can do so by explaining further the input sentence, using more concise wording, adding qualifiers and auxiliary information etc.",
```
Comment (danyaljj) [inline tasks/task955_wiki_auto_style_transfer.json]: "using more concise wording" sounds like the opposite of what we want.

**Comment by danyaljj** [comment]
Left one comment; otherwise, it looks alright.

File: tasks/task955_wiki_auto_style_transfer.json (hunk 1 above)
Comment (garyhlai) [reply tasks/task955_wiki_auto_style_transfer.json]: ohhh it should be "more precise"
